trk/src/Screens/ClimbDetail/index.js

```diff
@@ -1,32 +1,74 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect, useContext } from 'react';
 import { Alert } from 'react-native';
 import TapsApi from '../../api/TapsApi';
+import ClimbsApi from '../../api/ClimbsApi';
 
 import { View, Text, StyleSheet, ScrollView, SafeAreaView, Image, TextInput, Button, TouchableWithoutFeedback, Keyboard, Share, TouchableOpacity } from 'react-native';
 
 import storage from '@react-native-firebase/storage';
 import SegmentedControl from '@react-native-segmented-control/segmented-control';
+import { AuthContext } from '../../Utils/AuthContext';
```

Comment:
used to need it in home but now we need it in climbdetail to add taps only if the user is not the setter of that climb